### ISSUE TITLE
Upgraded Legendary Weapons Change back to Original Form after Battle.

### ISSUE THREAD

**Issue by Aelyat** [sayucchin/P2-EP-PSP#12]
Hi! I have been playing with the english patch for a while and I got my Legendary Handgun for Maya, I don't know why but after spreading rumors that the legendary handgun is powerful or/and has "dark power" it lasts only for one battle after the rumour is spread and the weapon changed. I tested with Dead End and Megido Fire (the upgraded versions via rumors of the Legendary Gun) and after one battle it reverts back to Legendary Handgun by default. I noticed that the game somehow registers that the weapon is changed, altought at the Equip Screen it says it's the default weapon(not changed)... and when I try to spread the upgrade rumor again, it shows the option to turn the weapon "Ordinary" and when I choose this, it changes to either Dead End or Megido Fire (but it lasts only one battle and then it goes back to default not changed legendary handgun). 
I don't know if this has been reported, but I think there is a bug with the Legendary Handgun at least. I don't think the weapon should change to a upgraded version by choosing to spread that the legendary weapon is "ordinary". And also, the issue that the rumor effect only lasts for one battle and then after it reverts back to default. Anyway, just thought I should make a thread about it. And If necessary, I can upload some pictures later showing this issue.

**Comment by sayucchin** [comment]
When you say it only lasts one battle... did you contact an enemy make it run away in fear? Cos that's supposed to change the weapon. Does it say "toy gun"? Or does it consistently switch back to "legendary handgun" from "Megido Fire" after every encounter?

**Comment by Aelyat** [comment]
It's not the toy gun that has 1atk power. It's the default one you get when crafting, Legendary Handgun. If i try to make Megido Fire or Dead End it works but it lasts only 1 battle. After another battle(either if I win it or escape) it reverts to Legendary Handgun. And when I try to spread the rumor again it shows the option to turn it 'ordinary'. But the weapon reverted back already to Ordinary by it's own, it's a really weird issue. It's like the game registers that you spread the rumor for either dead end or megido fire, but at Equipment... it shows the Legendary W. with 160atk power and no elemental affinities. 

Edit: I just managed to permanent change the legendary weapon somehow by contact a enemy, spread the rumor and escape battle and then contact again, save it and re-load the game. Now the Megido Fire does not revert back... I don't know what or how I did it LOL
I imagine I did somehow. But i did not test with the other legendary weapons, it might also have this glitch. When I manage to craft the others I will see if does the same thing and keep you posted.

**Comment by sayucchin** [comment]
that is one interesting glitch. I've never encountered it myself after using all the legendary weapons extensively in the ex dungeon. I wonder if it was a visual glitch? Did you try using it when it's in ordinary mode against enemies that reflect physical? Did it reflect or was the weapon secretly doing elemental damage as megido fire even though the UI showed legendary handgun?

**Comment by Aelyat** [comment]
> that is one interesting glitch. I've never encountered it myself after using all the legendary weapons extensively in the ex dungeon. I wonder if it was a visual glitch? Did you try using it when it's in ordinary mode against enemies that reflect physical? Did it reflect or was the weapon secretly doing elemental damage as megido fire even though the UI showed legendary handgun?

I loaded my previous save file with this issue and it really does revert back to original Legendary Handgun. I tried Megido Fire (that has almighty element) on a enemy who was weak to gun, and the Weak pop-up did not show. Then... I escaped battle and tried on the same enemy (Now the weapon reverted to original form with Gun type) and the Weak pop up showed this time on Hit, as the enemy was weak to Gun. After that, I spread the avaliable rumor again to change to Megido Fire for testing once more and the result was the same. I am testing this out at B1 of Mt. Iwato btw, for reference.

**Comment by sayucchin** [comment]
can you email me your save file + system save data?
sayucchi.msg@gmail.com

**Comment by Aelyat** [comment]
I can't seem to transfer the paste only the files. So here you go, those
files are for the System Save data. (ULJM060810000)
ULJM060810000

Em ter., 28 de mar. de 2023 às 13:08, sayucchin ***@***.***>
escreveu:

**Comment by Aelyat** [comment]
This is one of the save files With Dead End weapon.
> ULJM060810067

Em ter., 28 de mar. de 2023 às 13:26, Ae'lyat Aguiar ***@***.***>
escreveu:

**Comment by Aelyat** [comment]
Those ones are with Megido Fire weapon. ULJM060810068

Em ter., 28 de mar. de 2023 às 13:27, Ae'lyat Aguiar ***@***.***>
escreveu:

**Comment by Aelyat** [comment]
Just uploaded. 

> can you email me your save file + system save data? [sayucchi.msg@gmail.com](mailto:sayucchi.msg@gmail.com)

**Comment by sayucchin** [comment]
thanks I'll have a look and see if we can replicate it

**Comment by sayucchin** [comment]
Okay I've looked at the save files but I can't replicate it from the earlier save file. It's as if the "ordinary" rumor has already been spread on your file and after the one-combat encounter delay the gun then switches back into normal mode in the inventory. Do you have any mods or cheats on? Did you spread two weapon rumors back to back without waiting for the 1 combat encounter delay to finish?

**Comment by Aelyat** [comment]
No, I don't have any other mods or cheats enabled with that save, apart
from the english translation patch. About the rumors, I spread just one at
a time.

Em sáb., 1 de abr. de 2023 às 11:31, sayucchin ***@***.***>
escreveu:
